tensorflow/tools/ci_build/install/install_pip_packages.sh

```diff
@@ -28,33 +28,33 @@ pip2 install wheel
 pip3 install wheel
 
 # Install six.
-pip2 install --upgrade six==1.10.0
-pip3 install --upgrade six==1.10.0
+pip2 install --upgrade six>=1.10.0
+pip3 install --upgrade six>=1.10.0
 
 # Install werkzeug.
-pip2 install --upgrade werkzeug==0.11.10
-pip3 install --upgrade werkzeug==0.11.10
+pip2 install --upgrade werkzeug>=0.11.10
+pip3 install --upgrade werkzeug>=0.11.10
 
 # Install bleach. html5lib will be picked up as a dependency.
 pip2 install --upgrade bleach==1.5.0
 pip3 install --upgrade bleach==1.5.0
 
 # Install markdown.
-pip2 install --upgrade markdown==2.6.8
-pip3 install --upgrade markdown==2.6.8
+pip2 install --upgrade markdown>=2.6.8
+pip3 install --upgrade markdown>=2.6.8
 
 # Install protobuf.
-pip2 install --upgrade protobuf==3.2.0
-pip3 install --upgrade protobuf==3.2.0
+pip2 install --upgrade protobuf>=3.2.0
+pip3 install --upgrade protobuf>=3.2.0
 
 # Remove obsolete version of six, which can sometimes confuse virtualenv.
 rm -rf /usr/lib/python3/dist-packages/six*
 
 # numpy needs to be installed from source to fix segfaults. See:
 # https://github.com/tensorflow/tensorflow/issues/6968
 # This workaround isn't needed for Ubuntu 16.04 or later.
-pip2 install --no-binary=:all: --upgrade numpy==1.12.0
-pip3 install --no-binary=:all: --upgrade numpy==1.12.0
+pip2 install --no-binary=:all: --upgrade numpy>=1.11.0
+pip3 install --no-binary=:all: --upgrade numpy>=1.11.0
```

Comment:
I think those should be at least in sync with the [setup.py](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/pip_package/setup.py)